rfc8446.xml

```diff
@@ -7538,6 +7547,20 @@ Nikos. Anyway, this is an IESG matter and you are AD, so perhaps you can
 resolve it or talk to Kathleen. -->
 
 </section>
+
+<section anchor="sharing-psks" title="Sharing PSKs">
+<t>
+TLS 1.3 takes a conservative approach to PSKs by binding them to a
+specific KDF. TLS 1.2 allows PSKs to be used with any hash function
+and the TLS 1.2 PRF. The constructions in TLS 1.2 and TLS 1.3,
+although both based on HMAC, are very different and there is no known
+way in which reuse of the same PSK in TLS 1.3 and TLS 1.2 would
+produce related output, although only limited formal analysis has been
+done. Implementations MAY wish to avoid reuse of keys between TLS 1.3
```

Comment:
I don't know that "MAY wish to" is really the right thing, here.  I could see a lowercase "may", or possibly "SHOULD avoid reuse [...] until a key derivation scheme such as [SHAREDPSK] or [UNIVERSAL-PSK] is agreed upon", since we think the message is "don't reuse until we have a plan", and there is real harm from having multiple different schemes get deployed.

Reply:
👍 I'd opt to make this a SHOULD.